README.md

```diff
@@ -6,9 +6,13 @@ This repository contains the source for the Ruby runtime for
 [Google App Engine Flexible](https://cloud.google.com/appengine/docs/flexible/).
 It comprises:
 
-* The base image `gcr.io/google-appengine/ruby` in the `appengine` directory.
+* The base image in the `appengine` directory.
 * The Ruby runtime build pipeline in the `builder` directory.
 
+The image can be found at `l.gcr.io/google/ruby`
```

Comment:
This repo includes a number of images, including this one plus all the runtime builder images, plus (in the future) the jruby runtime and other related runtimes. Can we move this text into appengine/README instead, to avoid confusion about which image we are talking about?

Note that the catalog will link to appengine/README for the ruby runtime base image (not this root README).